packages/storybook/src/nimble/chip/chip.stories.ts

```diff
@@ -23,11 +23,11 @@ interface ChipArgs {
 // prettier-ignore
 const metadata: Meta<ChipArgs> = {
     title: 'Components/Chip',
-    render: createUserSelectedThemeStory(html`
+    render: createUserSelectedThemeStory(html<ChipArgs>`
     ${disableStorybookZoomTransform}
-        <${chipTag} appearance="${x => x.appearance}" ?removable="${x => x.removable}" ?disabled="${x => x.disabled}">
-            ${x => x.content}
-            ${when(x => x.icon, html`
+        <${chipTag} appearance="${(x: ChipArgs) => x.appearance}" ?removable="${(x: ChipArgs) => x.removable}" ?disabled="${(x: ChipArgs) => x.disabled}">
```

Comment:
Are these changes necessary? Our other stories don't need to explicitly specify the type on every usage of the component variable (e.g. search for `x.appearance` in other stories.ts files).